migrations/20200507120000-create-comment-reactions.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, DataTypes) => {
+    await queryInterface.createTable('CommentReactions', {
+      id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true,
+      },
+      UserId: {
+        type: DataTypes.INTEGER,
+        references: { key: 'id', model: 'Users' },
+        onDelete: 'NO ACTION',
+        onUpdate: 'CASCADE',
+        allowNull: false,
+      },
+      CommentId: {
+        type: DataTypes.INTEGER,
+        references: { key: 'id', model: 'Comments' },
+        onDelete: 'NO ACTION',
```

Comment:
We can delete reactions if the comment gets deleted

```suggestion
        onDelete: 'CASCADE',
```